test/integration/targets/collections/collection_root_user/ansible_collections/testns/testcoll/meta/routing.yml

```diff
@@ -0,0 +1,37 @@
+plugin_routing:
+  action:
+    uses_redirected_action:
+      redirect: testns.testcoll.subclassed_normal
+  connection:
+    redirected_local:
+      redirect: ansible.builtin.local
+  modules:
+    multilevel1:
+      redirect: testns.testcoll.multilevel2
+    multilevel2:
+      redirect: testns.testcoll.multilevel3
+    multilevel3:
+      redirect: testns.testcoll.ping
+    uses_redirected_action:
+      redirect: ansible.builtin.ping
+    setup.ps1: ansible.windows.setup
+    looped_ping:
+      redirect: testns.testcoll.looped_ping2
+    looped_ping2:
+      redirect: testns.testcoll.looped_ping
+    bogus_redirect:
+      redirect: bogus.collection.shouldbomb
+    deprecated_ping:
+      deprecation:
+        removal_date: 2020-12-31
+        warning_text: old_ping will be removed in a future release of this collection. Use new_ping instead.
+    foobar_facts:
+      redirect: foobar_info
+    aliased_ping:
+      redirect: ansible.builtin.ping
+    dead_ping:
+      tombstone:
+        removal_date: 2019-12-31
+        warning_text: dead_ping has been removed
+
+requires_ansible: '>=2.11'
```

Comment:
Should this really be part of the routing file? I would have thought it should be it's own file or maybe in main.yml

Reply:
I had similar thoughts (hence my mention above of bikeshedding on locations/names)- this one was originally `runtime.yml` which kinda made more sense, since it's all stuff we need at or immediately after load-time.

Reply:
to keep the tradition meta/main.yml ?